sunpy/visualization/animator/wcs.py

```diff
@@ -60,9 +61,13 @@ class ArrayAnimatorWCS(ArrayAnimator):
        The yaxis label to use when drawing a line plot. Setting the label on
        the y-axis on an image plot should be done via ``coord_params``.
 
+    clip_interval : two-element `~astropy.units.Quantity`, optional
```

Comment:
Does this take any quantity?

Reply:
Percentage right? I assumed that's how it's supposed to be written.

Reply:
Sure it is the type but since it has to be a percent quantity, I wonder if its worth changing to that.

Reply:
All other docstrings in sunpy have ```astropy.units.Quantity``` regardless of them only working with one particular type though

Reply:
Leave it then.